BitwardenShared/Core/Vault/Repositories/VaultRepository.swift

```diff
@@ -1448,21 +1456,23 @@ extension DefaultVaultRepository: VaultRepository {
     /// Creates a `VaultListItem` from a `CipherView` with Fido2 credentials.
     /// - Parameter cipher: Cipher from which create the item.
     /// - Returns: The `VaultListItem` with the cipher and Fido2 credentials.
-    func createFido2VaultListItem(from cipher: CipherView) async throws -> VaultListItem? {
-        let decryptedFido2Credentials = try await clientService
-            .platform()
-            .fido2()
-            .decryptFido2AutofillCredentials(cipherView: cipher)
-
-        guard let fido2CredentialAutofillView = decryptedFido2Credentials.first else {
-            errorReporter.log(error: Fido2Error.decryptFido2AutofillCredentialsEmpty)
-            return nil
-        }
-
-        return VaultListItem(
-            cipherView: cipher,
-            fido2CredentialAutofillView: fido2CredentialAutofillView
-        )
+    func createFido2VaultListItem(from cipher: CipherListView) async throws -> VaultListItem? {
+        nil
+        // let decryptedFido2Credentials = try await clientService
+        //    .platform()
+        //    .fido2()
+        //    .decryptFido2AutofillCredentials(cipherView: cipher)
```

Comment:
@Hinton it looks like we'll need to have `decryptFido2AutofillCredentials(cipherView:)` updated to accept a `CipherListView` here unless I am missing something.